src/app.ts

```diff
@@ -27,6 +27,13 @@ function startApiServer() {
 
 
   app.listen(port);
+
+  await RootTypeInitializer.initializeRootTypes();
+  try {
```

Comment:
Do we need a try catch block here?
Does the function throw errors that need to be catched?
Would be nice to keep this file short.

Reply:
looks like we don't need it anymore after the refactoring. So I removed it.